xarray/core/common.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import absolute_import, division, print_function
 
+import numbers
 import warnings
 
 import numpy as np
 import pandas as pd
 
 from . import dtypes, formatting, ops
-from .pycompat import OrderedDict, basestring, dask_array_type, suppress
+from .arithmetic import SupportsArithmetic
+from .options import OPTIONS
```

Comment:
is `OPTIONS` ever used or is it needed in this scope for some reason?

Reply:
Oops we don't need it here.